velox/functions/prestosql/ArrayIntersectExcept.cpp

```diff
@@ -622,7 +622,10 @@ std::shared_ptr<exec::VectorFunction> createTypedArraysIntersectExcept(
   VELOX_CHECK_EQ(inputArgs.size(), 2);
   const BaseVector* rhs = inputArgs[1].constantValue.get();
 
-  if (elementType->providesCustomComparison()) {
+  if constexpr (kind == TypeKind::UNKNOWN) {
```

Comment:
UNKNOWN values can only be nulls. Intersecting arrays of NULLs can be done faster by looking at just the array sizes, no?

This change affects array_except function as well? If so, can we add test coverage?